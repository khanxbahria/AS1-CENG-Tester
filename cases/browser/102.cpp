#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("x", "o","w");
Tab B = Tab("g", "yyy","u");
Tab C = Tab("kkk", "rr","kk");
Tab D = Tab("z", "i","f");
Tab E = Tab("kkk", "ccc","nn");
Tab F = Tab("www", "bbb","aa");
Tab G = Tab("ddd", "ii","v");
Tab H = Tab("zz", "cc","rr");
Tab I = Tab("ss", "uuu","ll");
Tab J = Tab("f", "ii","gg");
Tab K = Tab("bb", "aaa","l");
Tab L = Tab("uuu", "tt","ww");
Tab M = Tab("cc", "fff","m");
Tab N = Tab("nn", "ee","gg");
Tab O = Tab("bbb", "q","o");
Tab P = Tab("o", "hhh","qq");
Tab Q = Tab("kkk", "jjj","ll");
Tab R = Tab("zz", "v","ll");
Tab S = Tab("ss", "ddd","mm");
Tab T = Tab("u", "mm","ll");
Tab U = Tab("l", "e","gg");
Tab V = Tab("tt", "l","e");
Tab W = Tab("uuu", "ww","gg");
Tab X = Tab("d", "c","ggg");
Tab Y = Tab("f", "zzz","oo");
Tab Z = Tab("hhh", "ww","uu");
a.changeActiveTabTo(0);
b.newTab(A);
c.newTab(B);
d.moveActiveTabTo(0);
a.newTab(C);
b.changeActiveTabTo(4);
c.newTab(D);
d.newTab(E);
a.newTab(F);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.newTab(G);
a.moveActiveTabTo(2);
b.newTab(H);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.newTab(I);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.newTab(J);
a.newTab(K);
b.newTab(L);
c.newTab(M);
d.moveActiveTabTo(1);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.newTab(N);
d.newTab(O);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.newTab(P);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.newTab(Q);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.moveActiveTabTo(2);
a.moveActiveTabTo(4);
b.newTab(R);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.newTab(S);
b.moveActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
a.newTab(T);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.newTab(U);
b.changeActiveTabTo(1);
c.newTab(V);
d.newTab(W);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.newTab(X);
d.moveActiveTabTo(3);
a.changeActiveTabTo(1);
b.newTab(Y);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.newTab(Z);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
d.newTab(Z);
a.moveActiveTabTo(4);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
c.moveActiveTabTo(1);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
d.changeActiveTabTo(1);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.changeActiveTabTo(4);
b.moveActiveTabTo(5);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(0);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
a.changeActiveTabTo(4);
d.moveActiveTabTo(4);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
a.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(2);
d.moveActiveTabTo(2);
a.moveActiveTabTo(2);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(2);
c.changeActiveTabTo(2);
a.moveActiveTabTo(5);
b.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(0);
c.moveActiveTabTo(0);
d.changeActiveTabTo(0);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(0);
c.moveActiveTabTo(0);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.changeActiveTabTo(4);
b.changeActiveTabTo(0);
d.moveActiveTabTo(0);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
d.moveActiveTabTo(4);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
a.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->switchToWindow(9);
brows->closeEmptyWindows();
brows->closeWindow();
brows->print();
brows->switchToWindow(2);
brows->newWindow();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->print();
brows->newWindow();
brows->getWindow(7);
brows->print();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->getWindow(3);
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->getWindow(8);
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->switchToWindow(6);
brows->print();
brows->closeWindow();
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(8);
brows->closeWindow();
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->newWindow();
brows->getWindow(8);
brows->closeWindow();
brows->getWindow(5);
brows->print();
brows->print();
brows->print();
brows->print();
brows->closeWindow();
brows->closeWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->newWindow();
brows->closeEmptyWindows();
brows->print();
brows->print();
brows->closeWindow();
brows->switchToWindow(4);
brows->newWindow();
brows->closeEmptyWindows();
brows->newWindow();
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(8);
brows->closeWindow();
brows->getWindow(8);
brows->print();
brows->newWindow();
brows->closeWindow();
brows->print();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeWindow();
brows->newWindow();
brows->print();
brows->closeWindow();
brows->switchToWindow(4);
brows->switchToWindow(0);
brows->mergeAllWindows();
brows->getWindow(0);
brows->switchToWindow(4);
brows->mergeAllWindows();
brows->switchToWindow(5);
brows->closeWindow();
brows->mergeAllWindows();
brows->getWindow(0);
brows->getWindow(6);
brows->newWindow();
brows->print();
brows->switchToWindow(1);
brows->newWindow();
brows->getWindow(7);
brows->newWindow();
brows->closeWindow();
brows->closeWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->print();
brows->closeEmptyWindows();
brows->print();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->print();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->print();
brows->newWindow();
brows->newWindow();
brows->switchToWindow(0);
brows->mergeAllWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->switchToWindow(0);
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->getWindow(4);
brows->switchToWindow(0);
brows->newWindow();
brows->print();
brows->newWindow();
brows->print();
brows->closeWindow();
brows->switchToWindow(5);
brows->newWindow();
brows->mergeAllWindows();
delete brows;
}
	return 0;
}