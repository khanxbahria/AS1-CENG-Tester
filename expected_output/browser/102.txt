The active tab is tab 1
rr
kkk
kk

ddd
ss
mm

mm
u
ll

ww
hhh
uu

e
l
gg

bbb
www
aa

uuu
ss
ll

aaa
bb
l

jjj
kkk
ll





hhh
o
qq

The active tab is tab 0




o
x
w

zzz
f
oo

tt
uuu
ww

cc
zz
rr

v
zz
ll

The active tab is tab 1
l
tt
e

fff
cc
m

c
d
ggg

ee
nn
gg





yyy
g
u

i
z
f

The active tab is tab 1




ww
hhh
uu

ii
f
gg

ww
uuu
gg

ccc
kkk
nn

q
bbb
o

ii
ddd
v

The active tab is tab 31




l
tt
e

fff
cc
m

c
d
ggg

ee
nn
gg





yyy
g
u

i
z
f

rr
kkk
kk

ddd
ss
mm

mm
u
ll

ww
hhh
uu

e
l
gg

bbb
www
aa

uuu
ss
ll

aaa
bb
l

jjj
kkk
ll





hhh
o
qq





o
x
w

zzz
f
oo

tt
uuu
ww

cc
zz
rr

v
zz
ll





ww
hhh
uu

ii
f
gg

ww
uuu
gg

ccc
kkk
nn

q
bbb
o

ii
ddd
v

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 31




l
tt
e

fff
cc
m

c
d
ggg

ee
nn
gg





yyy
g
u

i
z
f

rr
kkk
kk

ddd
ss
mm

mm
u
ll

ww
hhh
uu

e
l
gg

bbb
www
aa

uuu
ss
ll

aaa
bb
l

jjj
kkk
ll





hhh
o
qq





o
x
w

zzz
f
oo

tt
uuu
ww

cc
zz
rr

v
zz
ll





ww
hhh
uu

ii
f
gg

ww
uuu
gg

ccc
kkk
nn

q
bbb
o

ii
ddd
v

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 32








l
tt
e

fff
cc
m

c
d
ggg

ee
nn
gg





yyy
g
u

i
z
f

rr
kkk
kk

ddd
ss
mm

mm
u
ll

ww
hhh
uu

e
l
gg

bbb
www
aa

uuu
ss
ll

aaa
bb
l

jjj
kkk
ll





hhh
o
qq





o
x
w

zzz
f
oo

tt
uuu
ww

cc
zz
rr

v
zz
ll





ww
hhh
uu

ii
f
gg

ww
uuu
gg

ccc
kkk
nn

q
bbb
o

ii
ddd
v

The active tab is tab 0




The active tab is tab 32








l
tt
e

fff
cc
m

c
d
ggg

ee
nn
gg





yyy
g
u

i
z
f

rr
kkk
kk

ddd
ss
mm

mm
u
ll

ww
hhh
uu

e
l
gg

bbb
www
aa

uuu
ss
ll

aaa
bb
l

jjj
kkk
ll





hhh
o
qq





o
x
w

zzz
f
oo

tt
uuu
ww

cc
zz
rr

v
zz
ll





ww
hhh
uu

ii
f
gg

ww
uuu
gg

ccc
kkk
nn

q
bbb
o

ii
ddd
v

Window is empty
The list is empty.
The active tab is tab 0




Window is empty
The list is empty.
The active tab is tab 0




Window is empty
The list is empty.
The active tab is tab 0




Window is empty
The list is empty.
The active tab is tab 0




Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




